packages/manager/apps/web/client/app/dns-zone/history/history.module.js

```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+import 'angular-translate';
+import componentDiff from './diff/history.component';
```

Comment:
Can you please rename the folder **diff** with something meaning full ?

Reply:
I prefer diff-tool-viewer if you agree of course (since dnz-zone-history could be a good name for the dashboard also and be misunderpreted)

Reply:
may be dns-zone-diff-tool-viewer ? I am okay with diff-tool-viewer as well

Reply:
dns-history-diff-tool-viewer is very clear